learningresources/tests/test_api.py

```diff
@@ -83,6 +90,11 @@ def test_get_resource(self):
             )
         )
 
+    def test_invalid_create_repo(self):
+        """Create a repository with empty name and description"""
+        with self.assertRaises(IntegrityError):
```

Comment:
I think we should capture integrity errors at the API level, and add a `ValidationError` to the `learningresources.api` custom exceptions. It may be out of scope for this PR. Any opinions on this @pdpinch @carsongee?